hibernate-reactive-core/src/test/java/org/hibernate/reactive/types/JavaTypesArrayTest.java

```diff
@@ -307,8 +432,18 @@ private static class Basic {
 		// the default column type when creating the schema is too small on some databases
 		@Column(length = 5000)
 		BigInteger[] bigIntegerArray;
+
+		BigDecimal[] bigDecimalArrayNoAnnotations; // FAILS:  DB2, MySQL, MariaDB, SqlServer
+
+		@Array(length = 5)
+		BigDecimal[] bigDecimalArrayWithArrayAnnotation; // FAILS:  DB2, MySQL, MariaDB, SqlServer
```

Comment:
Avoid this type of comments: if we fix it, then we have to look for comments around the code that are obsolete.
The test case is already disabled for these databases. There's no need to leave comments around.

Reply:
removed